src/gluonts/nursery/temporal_hierarchical_forecasting/train_and_eval_cop_deepar.py

```diff
@@ -22,6 +22,9 @@
 )
 
 
+EVALUATE_ALL_LEVELS = True
```

Comment:
Is this change unrelated?

Reply:
In the previous version this parameter was not defined, and it was leading to an error.

Reply:
@rshyamsundar If we keep `EVALUATE_ALL_LEVELS = False` then I think we can merge this quickly and independently from the PR https://github.com/awslabs/gluonts/pull/2944.